app/scripts/auto-type/index.js

```diff
@@ -58,8 +58,8 @@ const AutoType = {
         }
     },
 
-    runAndHandleResult(result) {
-        this.run(result, err => {
+    runAndHandleResult(result, windowId) {
```

Comment:
This function is called without `windowId` in one place in this file, looks like it will fail there.

Reply:
If I'm not mistaken (currently on my phone so can't check) that is the call for when a matching entry was found automatically, therefore the window to type to is already the active window and the id doesn't necessarily have to be passed.
I think I still tried to put it in there but I didn't have access to windowInfo, but maybe I'm just misremembering something.
If the Id really has to be passed there, I'll take a look at it in a few hours or something.

Reply:
Well, you can make it optional, but if so, you need to handle it [here](https://github.com/keeweb/keeweb/pull/1511/files#diff-e3232c8bbd07c6d2be79351d60d87a21R80), otherwise it will pass empty `--window` arg to `xdotool` and I guess, `xdotool` won't like this.

Reply:
Actually I just tested it and if you pass undefined as the window argument it just uses the active window, so I don't think it would really be necessary, if you still want it I can implement it.

Reply:
It's better not to pass `undefined`, this can have unexpected effects and it _looks like_ that the parameter is required. If it's optional, let's handle it in the key-emitter and explicitly remove it from the command line, instead of inserting `undefined` there.

Reply:
It should be fixed now.

Reply:
Awesome, thank you!